airbyte-db/lib/src/main/java/io/airbyte/db/instance/jobs/migrations/V0_29_15_001__Add_temporalWorkflowId_col_to_Attempts.java

```diff
@@ -38,7 +38,8 @@ public void migrate(Context context) throws Exception {
     // As database schema changes, the generated jOOQ code can be deprecated. So
     // old migration may not compile if there is any generated code.
     DSLContext ctx = DSL.using(context.getConnection());
-    ctx.alterTable("attempts").addColumn(DSL.field("temporal_workflow_id", SQLDataType.VARCHAR(256).nullable(true)))
+    ctx.alterTable("attempts")
+        .addColumnIfNotExists(DSL.field("temporal_workflow_id", SQLDataType.VARCHAR(256).nullable(true)))
```

Comment:
This is not necessary. But I feel that we should be extra cautious. This change makes the migration idempotent. Even if the user's database is in a weird state (e.g. this column is manually added), it will work.